### PR TITLE
[dotnet] - Support 9.0 version - changes

### PR DESCRIPTION
**Pull request by gauravsaini04**
Ref: [#1184](https://github.com/devcontainers/images/issues/1184)
**Devcontainer Name**
* DotNet

**Description**
* Changes for DotNet v9.0 added support

**_Changelog_***
* Changed corresponding files to reflect the above

**Checklist**
* [x] Checked that applied changes work as expected

### CODE REVIEW

File: src/dotnet-mssql/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ Develop C# and .NET Core based applications. Includes all needed SDKs, extension
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| imageVariant | .NET version: | string | 8.0-bookworm |
+| imageVariant | .NET version: | string | 9.0-bookworm |
```
Comment (samruddhikhandale) [inline src/dotnet-mssql/README.md:10]: Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!

File: src/dotnet-mssql/devcontainer-template.json [hunk 2]
```diff
@@ -11,17 +11,20 @@
...
             "proposals": [
+				"9.0",
                 "8.0",
                 "6.0",
+				"9.0-bookworm",
                 "8.0-bookworm",
                 "6.0-bookworm",
                 "6.0-bullseye",
                 "8.0-noble",
+				"9.0-jammy",
```
Comment (samruddhikhandale) [inline src/dotnet-mssql/devcontainer-template.json:22]: We don't support `jammy`, its `noble`

File: src/dotnet-mssql/devcontainer-template.json [hunk 3]
```diff
@@ -11,17 +11,20 @@
...
                 "8.0-jammy",
                 "6.0-jammy",
                 "6.0-focal"
             ],
-            "default": "8.0-bookworm"
+            "default": "9.0-bookworm"
```
Comment (samruddhikhandale) [inline src/dotnet-mssql/devcontainer-template.json:27]: Can you fix the tests?

File: src/dotnet-postgres/README.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ Develop C# and .NET Core based applications. Includes all needed SDKs, extension
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| imageVariant | .NET version: | string | 8.0-bookworm |
+| imageVariant | .NET version: | string | 9.0-bookworm |
```
Comment (samruddhikhandale) [inline src/dotnet-postgres/README.md:10]: Same comment

File: src/dotnet-postgres/devcontainer-template.json [hunk 5]
```diff
@@ -11,17 +11,20 @@
...
             "proposals": [
+				"9.0",
                 "8.0",
                 "6.0",
+				"9.0-bookworm",
                 "8.0-bookworm",
                 "6.0-bookworm",
                 "6.0-bullseye",
                 "8.0-noble",
+				"9.0-jammy",
```
Comment (samruddhikhandale) [inline src/dotnet-postgres/devcontainer-template.json:22]: Same comment

File: src/dotnet/README.md [hunk 6]
```diff
@@ -7,7 +7,7 @@ Develop C# and .NET based applications. Includes all needed SDKs, extensions, an
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| imageVariant | .NET version: | string | 8.0-bookworm |
+| imageVariant | .NET version: | string | 9.0-bookworm |
```
Comment (samruddhikhandale) [inline src/dotnet/README.md:10]: Same comment

File: src/dotnet/devcontainer-template.json [hunk 7]
```diff
@@ -11,17 +11,20 @@
...
             "proposals": [
+				"9.0",
                 "8.0",
                 "6.0",
+				"9.0-bookworm",
                 "8.0-bookworm",
                 "6.0-bookworm",
                 "6.0-bullseye",
                 "8.0-noble",
+				"9.0-jammy",
```
Comment (samruddhikhandale) [inline src/dotnet/devcontainer-template.json:22]: Same comment

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: src/dotnet-mssql/README.md [hunk 8]
```diff
@@ -7,7 +7,7 @@ Develop C# and .NET Core based applications. Includes all needed SDKs, extension
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| imageVariant | .NET version: | string | 9.0-bookworm |
+| imageVariant | .NET version: | string | 8.0-bookworm |
```
Comment (samruddhikhandale) [inline src/dotnet-mssql/README.md:10]: Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!

File: src/dotnet-postgres/README.md [hunk 9]
```diff
@@ -7,7 +7,7 @@ Develop C# and .NET Core based applications. Includes all needed SDKs, extension
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| imageVariant | .NET version: | string | 9.0-bookworm |
+| imageVariant | .NET version: | string | 8.0-bookworm |
```
Comment (samruddhikhandale) [inline src/dotnet-postgres/README.md:10]: Same comment here!

File: src/dotnet/README.md [hunk 10]
```diff
@@ -7,7 +7,7 @@ Develop C# and .NET based applications. Includes all needed SDKs, extensions, an
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| imageVariant | .NET version: | string | 9.0-bookworm |
+| imageVariant | .NET version: | string | 8.0-bookworm |
```
Comment (samruddhikhandale) [inline src/dotnet/README.md:10]: Same comment here!

File: src/dotnet-mssql/README.md (hunk 8 above)
Comment (gauravsaini04) [reply src/dotnet-mssql/README.md:10]: done

File: src/dotnet/README.md (hunk 10 above)
Comment (gauravsaini04) [reply src/dotnet/README.md:10]: done

File: src/dotnet-postgres/README.md (hunk 9 above)
Comment (gauravsaini04) [reply src/dotnet-postgres/README.md:10]: done

**Review by samruddhikhandale — APPROVED** [review]
